src/thx/Arrays.hx

```diff
@@ -622,6 +622,21 @@ It returns the last element of the array or null if the array is empty.
     return array[array.length-1];
 
 /**
+Static wrapper for `Array` `map` function.
+**/
+  #if js inline #end
```

Comment:
I'll go ahead and remove the native map usage... seems to be significantly slower than the for loop.